openapi.yaml

```diff
@@ -49,156 +95,494 @@ paths:
           schema:
             $ref: '#/definitions/ErrorResult'
+    patch:
+      summary: Update information of a specific park
+      tags: 
+        - parks
+      description: Update park by unique ID
+      operationId: updateParkById
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/ownerId'
+        - $ref: '#/parameters/ownerPassword'
+        - $ref: '#/parameters/parkIdPath'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/ParkResult'
+        '404':
+          description: Park resource not found
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+    delete:
+      summary: Delete a specific park
+      tags: 
+        - parks
+      description: Delete park by unique ID
+      operationId: deleteParkById
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/ownerId'
+        - $ref: '#/parameters/ownerPassword'
+        - $ref: '#/parameters/parkIdPath'
+      responses:
+        '204':
+          description: Park successfully deleted
+        '400':
+          description: Incorrect syntax in request, or required fields missing
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '404':
+          description: Park resource not found
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+  /owners:
+    get:
+      summary: Lists all park owners
+      tags:
+        - owners
+      description: Get all park owners
+      operationId: getOwner
+      parameters:
+      - $ref: '#/parameters/authorization'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/OwnersResult'
+        '400':
+          description: Incorrect syntax in request, or required fields missing
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '404':
+          description: Park resource not found
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+    post:
+      summary: Creates a new park owner account
+      tags:
+        - owners
+      description: Post a new park owner
+      operationId: postOwner
+      parameters:
+      - $ref: '#/parameters/authorization'
+      - in: query
+        name: Name / Organization
+        description: Your name or organization name
+        type: string
+        required: true
+      - in: query
+        name: Password
+        description: Enter a password to access your parks with
+        type: string
```

Comment:
You can also specify `password` format for it: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types